mpags-cipher.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+#include <string>
+#include <vector>
+
+int main(int argc, char* argv[])
+{
+    const std::vector<std::string> cmdLineArgs{argv,argv+argc};
+
+    for(std::size_t i{0};i<cmdLineArgs.size();i++)
+    { 
+
+        if(cmdLineArgs[i] == "-h" or cmdLineArgs[i] == "--help")
+        {
+            std::cout << "Help Text" << std::endl;
+            continue;
+        }
+        
+        if(cmdLineArgs[i] == "--version")
+        {
+            std::cout << "version 1.0.0" << std::endl;
+            continue;
```

Comment:
As with `--help`, the version option should also exit after giving the version string using `return`.